modules/articles/client/services/models/article.client.model.js

```diff
@@ -0,0 +1,19 @@
+(function () {
```

Comment:
In this case, angular calls it a factory, not service.  If we are going to have this rigid way of thinking, then menu's should be [menuRun](https://github.com/meanjs/mean/pull/1126/files#diff-7868497d6ee128c169faf0cb57b8f855R6)?

The style guide leaves this type of detail up to interpretation.